src/main/scala/uk/gov/hmrc/ct/ct600/v3/B325.scala

```diff
@@ -16,13 +16,19 @@
 
 package uk.gov.hmrc.ct.ct600.v3
 
-import uk.gov.hmrc.ct.box.{CtBoxIdentifier, CtInteger, Linked}
-import uk.gov.hmrc.ct.computations.CP117
+import uk.gov.hmrc.ct.box.{Calculated, CtBoxIdentifier, CtInteger}
+import uk.gov.hmrc.ct.ct600.v3.calculations.B325Calculator
+import uk.gov.hmrc.ct.ct600.v3.retriever.CT600BoxRetriever
 
 
 case class B325(value: Int) extends CtBoxIdentifier(name = "Northern Ireland profits included") with CtInteger
```

Comment:
Option was here also

Reply:
As per CT-Spec B325 needs to be populated or else it has to be 0.